contracts/deploy/041_stop_buyback_swaps.js

```diff
@@ -0,0 +1,24 @@
+const hre = require("hardhat");
+
+const addresses = require("../utils/addresses");
+const { deploymentWithProposal } = require("../utils/deploy");
+module.exports = deploymentWithProposal(
+  { deployName: "041_stop_buyback_swaps", forceDeploy: false },
+  async ({}) => {
+    // Buyback contract
+    const cBuyback = await ethers.getContract("Buyback");
+
+    // Governance proposal
+    return {
+      name: "Deploy all new contracts and migrate all funds",
```

Comment:
Update the proposal name to reflect what this deploy actually does.

Reply:
thanks fixed 👍